### PR TITLE
[Docs] [YSQL] Added a limitation for READ COMMITTED isolation level

### PR DESCRIPTION
**Pull request by pkj415**

### CODE REVIEW

File: docs/content/latest/architecture/transactions/read-committed.md [hunk 1]
```diff
@@ -1204,6 +1200,15 @@ This feature interacts with the following features:
 1. **Follower reads (integration in progress):** When follower reads is turned on, the read point for each statement in a READ COMMITTED transaction will be picked as _Now()_ - _yb_follower_read_staleness_ms_ (if the transaction/statement is known to be explicitly/ implicitly read only).
 2. **Pessimistic locking:** READ COMMITTED has a dependency on pessimistic locking to fully work. To be precise, on facing a conflict, a transaction has to wait for the conflicting transaction to rollback/commit. Pessimistic locking behaviour can be seen for READ COMMITTED. An optimized version of pessimistic locking will come in near future, which will give better performance and will also work for REPEATABLE READ and SERIALIZABLE isolation levels. The optimized version will also help detect deadlocks proactively instead of relying on statement timeouts for deadlock avoidance (see example 1).
 
+## Limitations
```
Comment (ddhodge) [inline docs/content/latest/architecture/transactions/read-committed.md:1203]: I wouldn't number any headings, but if you do, number this one as well

File: docs/content/latest/architecture/transactions/read-committed.md [hunk 2]
```diff
@@ -1204,6 +1200,15 @@ This feature interacts with the following features:
...
+
+1. READ COMMITTED semantics ensure that the client doesn't face conflict and read restart errors. YSQL maintains these semantics as long as a statement's output doesn't exceed ysql_output_buffer_size (a gflag with a default of 256KB). If this condition is not met, YSQL will resort to optimistic locking for that statement.
```
Comment (ddhodge) [inline docs/content/latest/architecture/transactions/read-committed.md:1205]: put the gflag ysql_output_buffer_size inside ` characters, like ysql_pg_conf_csv is in section 6

File: docs/content/latest/architecture/transactions/read-committed.md [hunk 3]
```diff
@@ -1204,6 +1200,15 @@ This feature interacts with the following features:
...
+
+2. PostgreSQL requires the following as mentioned in its docs [here](https://www.postgresql.org/docs/current/xfunc-volatility.html) - _"STABLE and IMMUTABLE functions use a snapshot established as of the start of the calling query, whereas VOLATILE functions obtain a fresh snapshot at the start of each query they execute."_. YSQL uses a single snapshot for the whole procedure instead of one for each
```
Comment (ddhodge) [inline docs/content/latest/architecture/transactions/read-committed.md:1207]: Remove extra period at end of first sentence, after the underline. There is a period in the quote, which finishes the sentence.

**Review by ddhodge — APPROVED** [review]
Minor comments

File: docs/content/latest/architecture/transactions/read-committed.md (hunk 1 above)
Comment (pkj415) [reply docs/content/latest/architecture/transactions/read-committed.md:1203]: Removed numbering

File: docs/content/latest/architecture/transactions/read-committed.md (hunk 2 above)
Comment (pkj415) [reply docs/content/latest/architecture/transactions/read-committed.md:1205]: done

File: docs/content/latest/architecture/transactions/read-committed.md (hunk 3 above)
Comment (pkj415) [reply docs/content/latest/architecture/transactions/read-committed.md:1207]: done

**Pull request merged**